addons/binding/org.openhab.binding.deconz/src/main/java/org/openhab/binding/deconz/internal/handler/SensorThingHandler.java

```diff
@@ -256,6 +257,11 @@ public void valueUpdated(String channelID, SensorState state) {
                     updateState(channelID, new DecimalType(status));
                 }
                 break;
+            case BindingConstants.CHANNEL_OPENCLOSE:
+                if (open != null) {
+                    updateState(channelID, OnOffType.from(open));
```

Comment:
```suggestion
                    updateState(channelID, open ? OpenClosedType.OPEN : OpenClosedType.CLOSED);
```

The following additional import is required for this
import org.eclipse.smarthome.core.library.types.OpenClosedType;

Reply:
done